start.sh

```diff
@@ -94,22 +95,27 @@ docker compose up -d --build
 if [ $? -eq 0 ]; then
     print_success "Docker Compose up succeeded."
 else
-    print_fail "Docker Compose up failed." >&2  # Send error message to stderr
+    print_fail "Docker Compose up failed." >&2 # Send error message to stderr
     exit 1
 fi
 
-###### 4.) Install NPM modules and then start the frontend in dev mode  ######
+###### 4.) Generate mock data
+cd ./dev_setup/mock_data
```

Comment:
Seems like the scripts aren't running properly via `start.sh` because the pods are still in the processes of starting up?
 
Trying an alternate route of just adding a yarn script to run scripts on demand `yarn seed`

Reply:
> Seems like the scripts aren't running properly via `start.sh` because the pods are still in the processes of starting up?
> 
> Trying an alternate route of just adding a yarn script to run scripts on demand `yarn seed`

Sounds good! 👍